docs/guides/configuration.md

```diff
@@ -944,6 +944,42 @@ This example demonstrates how to specify an incremental by time range model kind
 
 Learn more about specifying Python models at the [Python models concepts page](../concepts/models/python_models.md#model-specification).
 
+
+#### Model Naming
+
+The `model_naming` configuration controls if model names are inferred based on the schema directory structure. If `model_naming` is not defined or `infer_names` is set to false, the model names must be provided explicitly.
+
+With `infer_names` set to true, model names are inferred based on their path. For example, a model located at `/models/catalog/schema/model.sql` would be named `catalog.schema.model`. However, if a name is provided in the model definition, it will take precedence over the inferred name.
```

Comment:
```suggestion
With `infer_names` set to true, model names are inferred based on their path. For example, a model located at `models/catalog/schema/model.sql` would be named `catalog.schema.model`. However, if a name is provided in the model definition, it will take precedence over the inferred name.
```